unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -103,11 +104,18 @@ private void LoadUserProfile(UserProfile userProfile)
 
     private void LoadOwnedWereables(UserProfile userProfile)
     {
-        if (ownedWearablesAlreadyLoaded || ownedWearablesRemainingRequests <= 0 || string.IsNullOrEmpty(userProfile.userId))
+        // If there is more than 1 minute that we have checked the owned wearables, we try it again
+        // This is done in order to retrieved the wearables after you has claimed them
+        if ((Time.realtimeSinceStartup < lastTimeOwnedWearablesChecked+60 &&
```

Comment:
In order to avoid those explanatory comments, we can extract the condition to a self-explanatory method like

`private bool IsWearableUpdateInCooldown()
{
    return Time.realtimeSinceStartup < lastTimeOwnedWearablesChecked + 60
}`

also, the 60 could be a constant.